website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,528 @@
+---
+layout: docs
+page_title: Manage custom login options
+description: >-
+  Use custom login flows to control how clients can authenticate to Vault
+  through the GUI.
+---
+
+# Manage custom login options for the Vault GUI
+
+@include 'alerts/enterprise-and-hcp.mdx'
+
+Use login customizations to preferred authentication methods and control which methods display in the Vault GUI login form.
+
+<Tip>
+
+- To apply a login customization to the `root` namespace, pass an empty string: `namespace=""`.
+
+- **Login settings are inheritable by default**. Namespaces inherit login settings from their parent 
+unless inheritance is explicitly disabled. To scope a login setting to a specific namespace, 
+set: `disable_inheritance=true`.
+
+</Tip>
```

Comment:
I wasn't sure if this was the right format or location for this info, but felt it was important to call out somewhere.
<img width="716" alt="Screenshot 2025-06-02 at 10 26 24 AM" src="https://github.com/user-attachments/assets/e54a164c-3514-4ee6-99e3-14c6b32ed340" />